docs/quickstart/prerequisites.mdx

```diff
@@ -2,8 +2,15 @@
 title: "Prerequisites"
 sidebarTitle: "Prerequisites"
 ---
-Before installing Odigos, you need to set up some tools on your computer. These are one-time installations that prepare your computer for running Odigos.
 
+This guide is perfect for you if:
```

Comment:
"This guide" is only for the "Prerequisites" page, right?
As this will be the first page of the "quickstart", I think many people might think that this refer to the quickstart itself and might skip it